script/tool/lib/src/common.dart

```diff
@@ -140,6 +142,13 @@ bool isLinuxPlugin(FileSystemEntity entity, FileSystem fileSystem) {
   return pluginSupportsPlatform(kLinux, entity, fileSystem);
 }
 
+/// Throws a [ToolExit] with `exitCode` and log the `errorMessage` in red.
+void ThrowsToolExit({@required String errorMessage, int exitCode = 1}) {
```

Comment:
s/Throws/Throw/; this name sounds like a test function to validate that something throws a ToolExit.